### ISSUE TITLE
[Feature Request] Light Overlay

### ISSUE THREAD

**Issue by ThePhaseCat** [trainb0y/FabriHUD#1]
idk how to format this lol

Would it be possible to include a light overlay as maybe like a toggle for both a hud element and something visible on the ground (Red if mobs can spawn and a different color for safe spots?)

**Comment by trainb0y** [comment]
Yeah, I'd like to add this eventually for sure. For now I added a light element with [v2.0.0](https://github.com/trainb0y/FabriHUD/releases/tag/v2.0.0), but an in-world overlay would definitely be nice.
